connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorHeartbeatConfig.java

```diff
@@ -57,7 +59,7 @@ short heartbeatsTopicReplicationFactor() {
         return getShort(HEARTBEATS_TOPIC_REPLICATION_FACTOR);
     }
 
-    protected static final ConfigDef CONNECTOR_CONFIG_DEF = new ConfigDef(BASE_CONNECTOR_CONFIG_DEF)
+    protected static final ConfigDef HEARTBEAT_CONFIG_DEF = new ConfigDef()
```

Comment:
I think we can get away without introducing a utility method to merge `ConfigDefs` by replacing this constant (and others like it) with a method that adds the connector-specific configuration properties to an existing `ConfigDef`:

```java
    private static ConfigDef defineHeartbeatConfig(ConfigDef baseConfig) {
        return baseConfig
            .define(
                    EMIT_HEARTBEATS_ENABLED,
                    ConfigDef.Type.BOOLEAN,
                    EMIT_HEARTBEATS_ENABLED_DEFAULT,
                    ConfigDef.Importance.LOW,
                    EMIT_HEARTBEATS_ENABLED_DOC)
            // ...
```